src/main/scala/org/camunda/feel/impl/builtin/TemporalBuiltinFunctions.scala

```diff
@@ -98,4 +94,73 @@ class TemporalBuiltinFunctions(clock: FeelEngineClock) {
     ValDate(date.`with`(TemporalAdjusters.lastDayOfMonth()))
   }
 
+  private def timestampFunction = builtinFunction(
+    params = List.empty,
+    invoke = { case _ =>
+      val timestamp = clock.getCurrentTime.toEpochSecond
+      ValNumber(timestamp)
+    }
+  )
+
+  private def timestampFunction2 = builtinFunction(
+    params = List("datetime"),
+    invoke = {
+      case List(ValDateTime(datetime))        =>
+        val timestamp = datetime.toEpochSecond
+        ValNumber(timestamp)
+      case List(ValLocalDateTime(datetime))   =>
+        val timestamp = datetime.toInstant(ZoneOffset.UTC).getEpochSecond
+        ValNumber(timestamp)
+      case _                                  => ValNull
+
+    }
+  )
+
+  private def timestampFunction3 = builtinFunction(
+    params = List("timestamp","zoneId"),
```

Comment:
:question: Why do we need a `zoneId` parameter?

UNIX timestamps are always in UTC.